incubator/prometheus-postgresql-adapter/values.yaml

```diff
@@ -0,0 +1,42 @@
+## prometheus-postgresql-adapter image version
+## ref: https://hub.docker.com/r/timescale/prometheus-postgresql-adapter/tags/
+##
+
+extraArgs: {}
+
+extraEnv: {}
+
+replicaCount: 1
+
+image:
+  repository: timescale/prometheus-postgresql-adapter
+  tag: 0.4
+  pullPolicy: IfNotPresent
+
+logLevel: info
+pgHost: postgres
```

Comment:
Would it be possible to make `pg` a dict and host, user, password, etc. keys?

Eg:
```
pg:
    host: postgres
    # user: postgres
    # password: ...
```

This would mirror the way it works for `image`, at least.

Reply:
@jasongwartz I followed the Postgres chart conventions that they used: https://hub.kubeapps.com/charts/bitnami/postgresql. I see what you're saying, but it's not atypical to have separate variables like I have.

Reply:
Yep, I see what you mean. Just thought it could help with the dot-name discussion above.

Reply:
No problem!

Reply:
While on the subject, should we make a PR for the adapter to accept env vars so we can use Secrets for the username and password?